sdk/python/tests/symbol/test_TransactionFactory.py

```diff
@@ -172,6 +172,26 @@ def test_can_autogenerate_mosaic_definition_id(self):
 		expected_id = generate_mosaic_id(factory.network.public_key_to_address(PublicKey(TEST_SIGNER_PUBLIC_KEY)), 123)
 		self.assertEqual(expected_id, transaction.id.value)
 
+	def test_can_autogenerate_mosaic_definition_flags(self):
+		# Arrange:
+		factory = self.create_factory()
+
+		# Act:
+		transaction = self.create_transaction(factory)({
+			'type': 'mosaic_definition_transaction_v1',
+			'signer_public_key': TEST_SIGNER_PUBLIC_KEY,
+			'nonce': 123,
+			'flags': 'supply_mutable restrictable transferable revokable'
+		})
+
+		# Assert:
+		self.assertEqual(type(transaction.flags), sc.MosaicFlags)
+		self.assertEqual(transaction.flags.value, 15)
```

Comment:
we generally have been putting expected value as lhs and actual value on rhs

for consistency, can you swap the arguments so expected values are first arguments?